### PR TITLE
sentiment.py and notebooks

### PR DESCRIPTION
**Pull request by prushto**
Bert analysis (sentiment.py) and a couple of simple notebooks added. No interaction with other files.

### CODE REVIEW

File: src/sentiment.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+device = torch.device("cuda")
+
+
+LR = 1e-5  # Learning rate
+
+
+class BERT_architecture(nn.Module):
+    def __init__(self, bert):
+        super(BERT_architecture, self).__init__()
```
Comment (reislerjul) [inline src/sentiment.py:22]: This architecture is more complicated than I wrote in the report! Please make sure the report mentions that your architecture uses 2 feed forward network with dropout and relu between the.

File: src/sentiment.py [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+        self.bert = bert
+        # dropout layer
+        self.dropout = nn.Dropout(0.2)
+        # relu activation function
+        self.relu = nn.ReLU()
+        # dense layer 1
+        self.fc1 = nn.Linear(768, 512)
+        # dense layer 2 (Output layer)
+        self.fc2 = nn.Linear(512, 5)  # 5 output classes
+        self.softmax = nn.LogSoftmax(dim=1)
```
Comment (reislerjul) [inline src/sentiment.py:32]: It looks like this is binary prediction. Let's also mention that in the report.

File: src/sentiment.py [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+
+    # freeze the BERT architecture
+    for param in bert.parameters():
+        param.requires_grad = False
+
+    # pass the pre-trained BERT to our define architecture
+    model = BERT_architecture(bert)
+
+    # push the model to GPU - I THINK DONT DO THIS
+    model = model.to(device)
```
Comment (reislerjul) [inline src/sentiment.py:237]: I think this is right

File: src/sentiment.py [hunk 4]
```diff
@@ -0,0 +1,302 @@
+# Code adapted from https://github.com/Himabindugssn/Sentiment-classification-using-transformers
```
Comment (reislerjul) [inline src/sentiment.py:1]: This code is awesome! Nice job.

**Review by reislerjul — COMMENTED** [review]
Looks great! Thanks for doing this. In the future I would just combine your two notebooks together. Most of the code is the same.

**Review by reislerjul — APPROVED** [review]

File: src/sentiment.py (hunk 3 above)
Comment (prushto) [reply src/sentiment.py:237]: Yeah it was just old comment i didn't delete

File: src/sentiment.py (hunk 4 above)
Comment (prushto) [reply src/sentiment.py:1]: Thanks... i mean i can't really take too much credit for it!
